pcsx2/gui/MainFrame.cpp

```diff
@@ -643,31 +643,65 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	if( cdvd )
+	if (cdvd2 == nullptr) // It doesn't point to anything when it's explicitly destroyed , let's insert it again.
 	{
-		if( vm )
-		{
-			cdvd->SetItemLabel(_("Reboot CDVD (full)"));
-			cdvd->SetHelp(_("Hard reset of the active VM."));
-		}
-		else
-		{
-			cdvd->SetItemLabel(_("Boot CDVD (full)"));
-			cdvd->SetHelp(_("Boot the VM using the current DVD or Iso source media"));
-		}	
+		// WX throws an assert when a non stock menu item is inserted without a label text as a parameter on Insert()
+		// Let's add a placeholder text just to avoid an assert.
+		cdvd2 = m_menuSys.Insert(1, MenuId_Boot_CDVD2, "Turtleli");
```

Comment:
Personally I was thinking of something like:
```
wxString full_label;
wxString full_help;
wxString fast_label;
wxString fast_help;

//
// Set label and help strings
//

cdvd->SetItemLabel(full_label);
cdvd->SetHelp(full_help);

if (source == CDVD_SourceType::CDVDsrc_NoDisc) {
	if (cdvd2)
		m_menuSys.Destroy(cdvd2);
} else {
	if (cdvd2) {
		cdvd2->SetItemLabel(fast_label);
		cdvd2->SetHelp(fast_help);
	} else {
		m_menuSys.Insert(1, MenuId_Boot_CDVD2, fast_label, fast_help);
	}
}
```
It'd save copy paste replication.